Makefile

```diff
@@ -105,6 +105,17 @@ CODER_ARCH_IMAGE_PREREQUISITES := \
 	build/coder_$(VERSION)_%.tar.gz
 endif
 
+# used to decide if we can build with boringcrypto
+ifeq ($(OS),Windows_NT)
+	local_os:=Windows
+	local_arch:="" #ignored, no boringcrypto support for Windows
+else
+	local_os:=$(shell uname -s)
+	local_arch:=$(shell uname -m)
+endif
+ifeq ($(local_arch),x86_64)
+	local_arch:=amd64
+endif
```

Comment:
You might be able to simplify this with `shell go env GOHOSTOS` and `GOHOSTARCH`

Reply:
Oh! That's much nicer